lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -0,0 +1,1124 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TupleSections #-}
+{-# LANGUAGE TypeApplications #-}
+
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+module Cardano.Wallet.Shelley.TransactionSpec
```

Comment:
Most of this module is irrelevant for Shelley since the transaction format has completely changed since Byron. I'd suggest to remove it; it'll be re-implement once we implement the transaction layer for Shelley.

Reply:
48cf00bd5